index.js

```diff
@@ -4,13 +4,16 @@
 module.exports = {
   name: 'ember-tether',
 
-  included: function(app) {
-    if (app.import) {
-      this.importBowerDependencies(app);
+  options: {
+    nodeAssets: {
+      tether: {
+        srcDir: 'dist',
+        import: ['js/tether.js']
+      }
     }
   },
 
-  importBowerDependencies: function(app) {
-    app.import(app.bowerDirectory + '/tether/dist/js/tether.js');
+  importBowerDependencies: function() {
```

Comment:
I originally trashed it, but this felt like it might give a better error message for anyone who blindly upgraded the deps in their addon. If you're not concerned about that case, I'm happy to throw it out.